credentialz/credentialz.proto

```diff
@@ -582,14 +585,36 @@ message RotateHostCredentialsResponse {
   }
 }
 
+// KeySuite specifies the valid types of keys and their lengths that
```

Comment:
this comment doesn't appear to match the content.

Reply:
Will fix the comment